admin_manual/configuration_server/occ_command.rst

```diff
@@ -1272,14 +1272,16 @@ units.
 Trashbin
 --------
 
-::
+These commands allow for manually managing various aspects of the trash bin (deleted files)::
 
  trashbin
-  trashbin:cleanup  [--all-users] [--] [<user_id>...]  Permanently remove deleted files
-  trashbin:restore  [--all-users] [--scope[=SCOPE]] [--since[=SINCE]] [--until[=UNTIL]] [--dry-run] [--] [<user_id>...]  Restore deleted files according to the given filters
+  trashbin:cleanup      Permanently remove deleted files
+  trashbin:expire       Expires the users trashbin
```

Comment:
The `expire` command is not really clear for me, you give a date or something after that it deletes the files?

Reply:
At present I'm just adding the missing command to the basic list that exists for each command per our current convention.

The description text there is pulled from the command output itself. Further details (the options) are available by running the command. Someone can do a follow-up PR in the spot down below that with examples if they want.

Reply:
Ok, I created a bug report for clarification: https://github.com/nextcloud/server/issues/45418